src/App.tsx

```diff
@@ -5,42 +5,61 @@ import {
 } from './components/Header/Header';
 import  Footer  from './components/Footer/Footer';
 import { HEADER_HEIGHT } from './components/Header/styles';
-import logo from './logo.svg';
 import './App.css';
+import axios, { AxiosError, AxiosResponse } from 'axios';
+import qs from 'qs';
 
-const token = async () => {
-  try {
-    const response = await fetch('https://api.particlespace.com/api/v1/authenticate', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/x-xxx-form-urlencoded',
-      },
-    } as RequestInit);
-    return response.json();
-  } catch (error) {
-    throw error;
-  }
-}
+/**
+ * Authorization header for the Particle Space API
+ */
+const data = qs.stringify({
+  'secret_key': process.env.SECRET_KEY,
+  'publish_key': process.env.PUBLISH_KEY,
+});
```

Comment:
These will need to be process.env.REACT_APP_PARTICLESPACE_PUBLISH_KEY just like in the .env.example file